modules/opts.nix

```diff
@@ -84,6 +84,6 @@ in
           ) optionsAttrs
         );
       in
-      mkIf (content != "") content;
+      mkIf (content != "") (mkOrder 300 content); # Move options to top of file below global table
```

Comment:
I think a priority after `500` may be better, to avoid surprising users who use `mkBefore`.

Reply:
(for another PR) This makes me think that we may want to introduce new `late` or `post` variants of the option that are inserted at the end of the file

Reply:
Updated to mkOrder 600 to not have a surprising breaking change for people who might have been using mkBefore with expectations no options are available in that scope.